src/components/RatingView.tsx

```diff
@@ -4,7 +4,7 @@ import { StyleSheet, Text, View } from 'react-native'
 const RatingView = ({ ratingName, ratingValue = 0 }: { ratingName: string; ratingValue?: number | null} ) => (
   <View style={styles.itemBackgroundStyle}>
     <View style={styles.ratingContainerStyle}>
-      <Text>{`${ratingName} Rating`}</Text>
+      <Text style={{fontWeight: 'bold'}}>{`${ratingName} Rating`}</Text>
```

Comment:
This style should be in the stylesheet, not the component.

Reply:
I didn't put it in the stylesheet since it. was only 1 style change but I can if you want me to.